ios/RCTConvert+BitmovinPlayer.swift

```diff
@@ -15,6 +15,48 @@ extension RCTConvert {
         if let licenseKey = json["licenseKey"] as? String {
             playerConfig.key = licenseKey
         }
+        if let playbackConfig = json["playbackConfig"] as? [String: Any?] {
+            /**
+             * Specifies whether autoplay is enabled.
+             *
+             * Default is `false`.
+             */
```

Comment:
Also this comment is not needed, as well as the other block comments in this function.